filament/backend/src/opengl/OpenGLProgram.cpp

```diff
@@ -356,24 +400,6 @@ void logCompilationError(io::ostream& out, Program::Shader shaderType,
     out << "Compilation error in " << to_string(shaderType) << " shader \"" << name << "\":\n"
         << "\"" << error << "\""
         << io::endl;
-
-    size_t lc = 1;
```

Comment:
Any reason to remove this? It's quite useful for debugging to see the line numbers and the shader source when we dump an error.

Reply:
Yes, if we want to keep it, we'll have to keep the source of all programs until we get at least to the first draw.
Even if we do this, then we'll have to free that memory on first draw, but that would be (potentially many) calls to free() -- which could block, etc...

Reply:
Could we keep it in debug builds?

Reply:
yeah we could.

Reply:
done